server/action_base.h

```diff
@@ -158,7 +159,7 @@ class Action {
  public:
   Action(std::shared_ptr<RequestObject> req, bool check_shutdown = true,
          std::shared_ptr<S3AuthClientFactory> auth_factory = nullptr,
-         bool skip_auth = false);
+         bool skip_auth = false, bool skip_authorization = true);
```

Comment:
Any particular reason of having skip_authorization as true by default

Reply:
Default value is needed because previous parameters have default values.
"true" allows to avoid any changes in motr_action_base.{h;cc} files